nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,412 @@
+package scala.scalanative.native
+package native
```

Comment:
Here you define double native, I'm not sure if you want to do this.
I think it should be either:
`package scala.scalanative.native`
or
```
package scala.scalanative
package native
```